browser/ui/views/tabs/vertical_tab_strip_browsertest.cc

```diff
@@ -529,11 +530,13 @@ IN_PROC_BROWSER_TEST_F(VerticalTabStripBrowserTest,
   AppendTab(browser());
   browser_view()->tabstrip()->StopAnimating(/* layout= */ true);
 
-  // When first tab is added, height should have tab's height plush top & bottom
+  // When first tab is added, height should have tab's height plus top & bottom
   // margin.
   contents_view_height +=
       (tabs::kVerticalTabHeight + tabs::kMarginForVerticalTabContainers * 2);
-  EXPECT_EQ(region_view->contents_view_->height(), contents_view_height);
+  ASSERT_TRUE(base::test::RunUntil([&]() {
```

Comment:
I wonder does it make sense to count `Layout` calls and check the number in tests to prevent the issues like this.
If so let's do this in follow up.

Reply:
Counting layout seems interesting. I'll try to figure out.